lib/zone.ts

```diff
@@ -519,6 +519,21 @@ interface EventTask extends Task {
   /* TS v1.8 => type: 'eventTask'; */
 }
 
+/**
+ * Extend the Error with additional fields for rewritten stack frames
+ */
+interface Error {
+  /**
+   * Stack trace where extra frames have been removed and zone names added.
+   */
+  zoneAwareStack?: string;
+
+  /**
+   * Original stack trace with no modiffications
+   */
+  originalStack?: string;
```

Comment:
It is the original Error.stack before it is rewritten to include zone information.